.travis/build.sh

```diff
@@ -17,7 +17,7 @@ pip install pytest-cov nbval
 pytest
 
 # check auto-gen files up-to-date
-python onnx/defs/gen_doc.py -o docs/Operators.md
+python onnx/defs/gen_doc.py
```

Comment:
There's also ci build setup for windows https://github.com/onnx/onnx/blob/master/appveyor.yml. which needs to be changed.

Reply:
Updated...